mxcube3/__init__.py

```diff
@@ -82,7 +82,7 @@ def complete_initialization(app):
         app.session = app.beamline.getObjectByRole("session")
         app.collect = app.beamline.getObjectByRole("collect")
         app.diffractometer = app.beamline.getObjectByRole("diffractometer")
-        if not hasattr(app.diffractometer, 'centring_motors_list'):
+        if getattr(app.diffractometer, 'centring_motors_list') is None:
```

Comment:
This is not good since `getattr` raises `AttributeError` exception if the designated
member does not exist. If you want to keep `getattr` put `try...except AttributeError` or just use `hasattr` ;) What's wrong why pep8 and `hasattr` by the way ?